guides/common/modules/proc_migrating-to-newly-deployed-satellite.adoc

```diff
@@ -0,0 +1,57 @@
+[id="migrating-to-newly-formed-{ProjectServer}]_{context}"]
+= Migrating to a Newly Formed {ProjectServer}
```

Comment:
Maybe simplify: `Migrating to a New {ProjectServer}`

Reply:
Plus adapt the ID and updated xrefs

Reply:
@Lennonka I'm a little confused about this...I did update the title of this one to be guides/common/modules/proc_migrating-to-a-new-server.adoc.  Any idea why this version is still available?